app/templates/src/static/css/cf-enhancements.less

```diff
@@ -0,0 +1,10 @@
+/* ==========================================================================
+   Capital Framework
+   Enhancements
+   ========================================================================== */
+
+// This is your project's cf enhancements.
+//
+// Whenever building enhancements or changes to existing CF components, write them
+// here to work out the details before contributing them back to Capital Framework
+// Project
```

Comment:
Weird line. I'd remove it. (And remember to add a blank line at the end of the file.)